schema_based_on_diagram.sql

```diff
@@ -0,0 +1,53 @@
+CREATE TABLE medical_histories (
+  id SERIAL PRIMARY KEY,
+  admitted_at TIMESTAMP NOT NULL,
+  patient_id INTEGER NOT NULL,
+  status VARCHAR(255) NOT NULL,
+  FOREIGN KEY (patient_id) REFERENCES patients(id)
+);
+
+CREATE TABLE patients (
+  id SERIAL PRIMARY KEY, 
+  name VARCHAR(250) NOT NULL, 
+  date_of_birth DATE) NOT NULL;
+
+CREATE TABLE invoices(
+  id SERIAL PRIMARY KEY, 
+  total_amount DECIMAL(5, 2) NOT NULL, 
+  generated_at TIMESTAMP NOT NULL, 
+  payed_at TIMESTAMP NOT NULL, 
+  medical_history_id INT NOT NULL
```

Comment:
- As is on the diagram, kindly add the FOREIGN KEY with REFERENCES missing on the table, with medical_histories(id) from the invoices table